packages/List/src/List.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import cx from 'classnames';
+
+import ListItem, { Option, Member, OptionType } from './ListItem';
+
+import './list.scss';
+
+export interface ListProps extends React.ComponentProps<'ul'> {
+  /** Add a specific class to the Select. */
+  className?: string;
+  /** Add a data-test tag for automated tests */
+  dataTest?: string;
+  /** The option that is currently being focused or hovered */
+  focusedOption?: OptionType | null;
+  /** True for a compact appearance. */
+  isCompact?: boolean;
+  /** The List gains checkboxes beside each option
+   * to be able to select multiple options */
+  multiple?: boolean;
+  /** Specify the text to display when there are no results found */
+  noResultsText?: string;
+  /** Called when an option becomes focused or hovered */
+  onOptionFocus?: (option: OptionType) => void;
+  /** Called when an option is selected */
+  onOptionChange?: (option: OptionType) => void;
+  /** A list of options */
+  options: OptionType[];
+  /** The initial selected option or a list of selected options. */
+  selectedOption?: OptionType | OptionType[] | null;
+}
+
+const List: React.FunctionComponent<ListProps> = (props: ListProps) => {
+  const {
+    className,
+    dataTest,
+    focusedOption,
+    isCompact = true,
```

Comment:
By default in Sigma the list is large, repack by false
```suggestion
    isCompact = false,
```